### ISSUE TITLE
JSON is broken

### ISSUE THREAD

**Issue by XSchelin** [PaperMtn/slack-watchman#47]
When exporting results to JSON, any message that itself has quote marks in results in broken JSON. For example:

```
{"timestamp": "2023-06-14 15:02:29,374", "level": "WORKSPACE", "message": "{"name": "Slack Team", "domain": "slack-team", "url": "https://slack-team.slack.com/", "email_domain": "", "is_verified": true, "discoverable": "open"}"}
```

What I'm hopeful for: that messages that are printed out to JSON escape quote marks before they're logged.

**Comment by PaperMtn** [comment]
I think this should only be for the User and Workspace output, and I think I probably know why. I will try to get a fix put in place

**Comment by PaperMtn** [comment]
This is fixed in [Version 4.0.2](https://github.com/PaperMtn/slack-watchman/releases/tag/4.0.2)
